{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\nallure_practice.py:19: Exception"}, "start": 1605527730760, "stop": 1605527730760, "uuid": "e9d837df-f259-4975-b50d-2963c9f85f90", "historyId": "725f28311fca5f30cd8879a814a1dbeb", "testCaseId": "aac35e6cd4194f20cb52472d4cd788f8", "fullName": "allure_practice#test_broken", "labels": [{"name": "suite", "value": "allure_practice"}, {"name": "host", "value": "LAPTOP-95H2SUTO"}, {"name": "thread", "value": "23628-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "allure_practice"}]}